websites/T/Teknofest/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "PXSTY",
+		"id": "571041849326698496"
+	},
+	"service": "Teknofest",
+	"description": {
+		"tr": "TEKNOFEST Havacılık, Uzay ve Teknoloji Festivali",
+		"en": "TEKNOFEST Havacılık, Uzay ve Teknoloji Festivali",
+		"nl": "TEKNOFEST Havacılık, Uzay ve Teknoloji Festivali"
+	},
+	"url": ["teknofest.org"],
+	"version": "1.0.12",
+	"logo": "https://i.imgur.com/b4It2CL.jpg",
+	"thumbnail": "https://i.imgur.com/b4It2CL.jpg",
```

Comment:
The thumbnail should be a screenshot or promotional banner, probably not the same as the logo.